Fix e surface integral normal and include multiplier vector (#77)
ECP-WarpX/artemis: Docs/source/usage/parameters.rst

           For example, we can define a surface on a y-plane at a location, `y_plane_location`, having a half cross-section
           from z=-Lz/2 to 0, where Lz is the length of the domain in the z-direction spanning from -Lz/2 to Lz/2, as follows:

           ``<reduced_diags_name>.reduced_function(x,y,z) = " (y > y_plane_location - dy/2.-epsilon) * (y < y_plane_location+epsilon) * (z > -Lz/2.) * (z < 0.+epsilon) * 1 "``
           

>            In this example, epsilon is very small number which is larger than machine precision.

Reviewer: ```suggestion
           In this example, epsilon is a very small number which is larger than machine precision.
```

Reviewer: Could we say that it needs to be a small fraction of the cell-size instead?

        * ``<reduced_diags_name>.surface_normal`` (`string`)
           The surface on which the surface integration is required. It must be either ``x``, ``y`` or ``z``.
           The direction of the normal is positive in the Cartesian directions.

>         * ``<reduced_diags_name>.multiplier_vector`` (`string`)

Reviewer: ```suggestion
        * ``<reduced_diags_name>.multiplier_vector`` (`string`) optional (default `1 1 1`)
```

Reviewer: Can we think of another name for the API instead of multiplier_vector? 
Possibly "scaling_factor"

>            This parameter is optional, is only used when the ``integration_type`` is ``surface``, and takes three values which are
>            multiplied to Ex, Ey, and Ez, respectively, before outputting the their surface integral. The default values are ``1 1 1``.
>            This parameter can be used in the following two scenarios:
>            Let's say, we require the surface integral of Ex on a surface having normal pointing in the negative x-direction.
>            In that case, we would specify the value of this parameter as ``-1 1 1`` so that the surface integral of Ex is multiplied by ``-1``.
>            As another example, we may require a line integral which is obtained by first taking surface integral
>            over a surface of height h and width w and then dividing by the width.
>            In this case we may specify the value of these parameters as ``1./w 1./w, 1./w``.
> 

Reviewer: ```suggestion
           This parameter is used when the ``integration_type`` is set to ``surface``. The parser takes three values to scale                      
           the reduced field quantities, namely, the surface integral of Ex, Ey, and Ez. 
           This parameter can be used in the following two scenarios:
           Let's say, we require the surface integral of Ex on a surface, with the surface normal in the negative x-direction.
           In that case, we would specify the value of this parameter as ``-1 1 1`` so that the surface integral of Ex is multiplied by ``-1``.
           As another example, we may require a line integral which is obtained by first taking surface integral
           over a surface of height h and width w and then dividing by the width.
           In this case, we may specify the value of these parameters as ``1./w 1./w, 1./w``. Note that this can only be done
           for a uniform grid simulation.

```
